packages/askui-nodejs/example_projects_templates/templates/askui-helper-windows.nj

```diff
@@ -17,12 +17,16 @@ beforeAll(async () => {
 });
 
 beforeEach(async () => {
+  /* Uncomment to enable video recording
   await aui.startVideoRecording();
+  */
```

Comment:
What do you think about adding a param to the askui init called `--with-video-recording` and set it to false as a default a value?

Reply:
I'm removing video recording for each customer. And nobody has complaint about it. 

The idea was to uncomment it, so the user has the possibility to enable it later.

Reply:
And for replacing the Lagacy Controller Video recording needs to be removed.

Reply:
Do you think we should merge this or should I add a flag? For me both are fine 😉 .

Reply:
Let's merge it.